l2geth/core/vm/ovm_state_manager.go

```diff
@@ -136,18 +136,7 @@ func putContractStorage(evm *EVM, contract *Contract, args map[string]interface{
 
 	// save the block number and address with modified key if it's not an eth_call
 	if evm.Context.EthCallSender == nil {
-		// save the value before
-		before := evm.StateDB.GetState(address, key)
 		evm.StateDB.SetState(address, key, val)
```

Comment:
Minor nit but this line can be pulled outside of the if/else (note the else below is just this line).

Reply:
Good catch